.github/workflows/ci.yml

```
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - *
```

This is invalid syntax.

```suggestion
    branches:
      - main
```

CI not running in #16 is because there are conflicts. You have to resolve conflicts first.

---

yeah sure. once this PR gets merged, I'll change the base to main and resolve the conflicts, and then run it again.